### DISCUSSION TITLE
run a cell periodically

### DISCUSSION THREAD

**Discussion by tachsin** [fonsp/Pluto.jl#1698] [Questions about using Pluto]
Hey there Plutorians!

I just started using Pluto and I'm already loving it! Now I have a situation that I'm running a simulation and it continuously dumps data inside a text file, and I read that data and plot through Pluto. 

I was wondering if there was a way to automate running the cell periodically so I could retire from clicking let's say at every 5 seconds?

Cheers!

**Comment by Pangoraw** [comment]
On the client side you can use `PlutoUI.Clock(..)` which creates a widgets that periodically updates its value. If you are interested in running dependent cells when the file changes, take a look at [this discussion](https://github.com/fonsp/Pluto.jl/discussions/1368#discussioncomment-1852458).
